### PR TITLE
Add CD action firebase #12

### PR DESCRIPTION
**Pull request by PiddePannkauga**

### CODE REVIEW

File: .github/workflows/client-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Firebase Deploy
+
+on:
+  push:
+    branches:
+      - master
+
+    paths:
+      - client/
```
Comment (Kalk88) [inline .github/workflows/client-deploy.yml]:
```suggestion
      - client/*
```

Comment (Kalk88) [inline .github/workflows/client-deploy.yml]:
```suggestion
      - client/*
```

**Comment by Kalk88** [comment]
The secret should be in place. :walking_man: :smirk_cat:

File: .github/workflows/client-deploy.yml [hunk 2]
```diff
@@ -40,4 +40,8 @@ jobs:
         with:
           args: deploy --only hosting:prod
         env:
+<<<<<<< HEAD
           FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+=======
+          FIREBASE_TOKEN: ${{ secrets.FIREBASE_TOKEN }}
+>>>>>>> fbdcc556319d6e2cb54e1787c42256b6628aee6b
```
Comment (Kalk88) [inline .github/workflows/client-deploy.yml]: Merge conflict needs to be resolved.

File: .github/workflows/client-deploy.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+      - name: Install Dependencies
+        run: npm install
+      - name: Build
+        run: npm run build-prod
```
Comment (Kalk88) [inline .github/workflows/client-deploy.yml]: I think we need to run this from the client folder.

File: .github/workflows/client-deploy.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+      - name: Install Dependencies
+        run: npm install
```
Comment (Kalk88) [inline .github/workflows/client-deploy.yml]: I think we need to run this from the client folder.

File: .github/workflows/client-deploy.yml (hunk 3 above)
Comment (Kalk88) [inline .github/workflows/client-deploy.yml]: We can't run this command from the root folder, and it should be npm run build. Check the package json in the client folder.

Comment (PiddePannkauga) [reply .github/workflows/client-deploy.yml]: "We can't run this command from the root folder"
What does this mean? Why could it not run from root of client?

Comment (PiddePannkauga) [reply .github/workflows/client-deploy.yml]: Pushed new change that I think does want you asked

Comment (Kalk88) [reply .github/workflows/client-deploy.yml]: We won't be standing in root of client

File: .github/workflows/client-deploy.yml [hunk 5]
```diff
@@ -18,7 +18,9 @@ jobs:
       - name: Install Dependencies
         run: npm install
```
Comment (Kalk88) [inline .github/workflows/client-deploy.yml]: I think it's the same here. but we can merge and see where it fails. =)

**Pull request merged**